components/plugin_marketplace/marketplace_item/marketplace_item.js

```diff
@@ -41,6 +41,61 @@ UpdateVersion.propTypes = {
     releaseNotesUrl: PropTypes.string,
 };
 
+// Label renders a tag showing a name and a description in a tooltip.
+// If a URL is provided, clicking on the tag will open the URL in a new tab.
+export const Label = ({name, description, url, color}) => {
+    const tag = (
+        <span
+            className='tag'
+            style={{backgroundColor: color || ''}}
+        >
+            {name.toUpperCase()}
```

Comment:
this will blow up if name is `null`.

Reply:
`name` is set as required in https://github.com/mattermost/mattermost-webapp/pull/4547/files/18096bccc7feeb1408a00095cf9ff928f979a5f9#diff-59f8fcb6440a639e363e470248fccd0dR93. Doesn't this mean it can't be `null`